pixeltable/func/udf.py

```diff
@@ -26,7 +26,8 @@ def udf(
         return_type: Optional[ts.ColumnType] = None,
         param_types: Optional[List[ts.ColumnType]] = None,
         batch_size: Optional[int] = None,
-        substitute_fn: Optional[Callable] = None
+        substitute_fn: Optional[Callable] = None,
+        _stored: bool = False
```

Comment:
Call this `_force_stored` for clarity (since the function might end up being stored even if it's False).